index.html

```diff
@@ -18,5 +18,129 @@
   <body>
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    <header>
+      <div class="logo">
+        <img src="./logo.png" alt="logo" />
+      </div>
+      <div class="title">
+        <h1>My Favourite YouTube Video Channels</h1>
+      </div>
+    </header>
+    <main>
+      <section class="start">
+        <div class="pagesImage">
+          <img src="./diy.png" alt="do it yourself image" />
+        </div>
+        <h2 class="title">How it all started?</h2>
+        <div class="paras">
```

Comment:
"paras" is not a particularly descriptive class name for this element - from what I can see the class is being applied to the content / copy of the page. Naming things in programming is notoriously tricky though!